packages/cascara/src/atoms/index.js

```diff
@@ -4,3 +4,4 @@
 
 export { default as Button } from './Button';
 export { default as Flex } from './Flex';
+export { default as FlexItem } from './Flex/FlexItem';
```

Comment:
I exported as an independent component because I was getting this error:
![image](https://user-images.githubusercontent.com/87443054/150014777-7dde6ee5-e83b-4f5b-8ddb-4c2b60bebe7d.png)

Reply:
@rociobaena we do not want to export the FlexItem directly as it can only be used as a child component of a Flex component. We should not be trying to do `<FlexItem>` as a component, but instead doing `<Flex.Item>` if we have already done an `import { Flex }`

Reply:
Brian, but having that, <Flex.Item> is why I have the code above. When I was debugging, seemed the name was causing the error. That's why I exported as FlexItem.

Reply:
I would need to see the MDX file that was causing this error then.

If true, we still do not want to export the FlexItem. Instead, we would possibly define FlexItem in the MDX_COMPONENTS file and map the props to a Flex.Item component. But we would need to see how the MDX files are getting processed and what it thinks the component node should be.

Reply:
I tried that but it didn't work, but... I'm thinking of something else. I'll keep you posted.

Reply:
Changes applied based on our Zoom converstation.